client/extensions/woocommerce/state/sites/coupons/test/handlers.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy, match } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import { WPCOM_HTTP_REQUEST } from 'state/action-types';
+import { fetchCouponsPage, couponsPageUpdated } from '../actions';
+import {
+	requestCouponsPage,
+	requestCouponsPageSuccess
+} from '../handlers';
+
+describe( 'handlers', () => {
```

Comment:
Please consider adding a test to ensure that the coupons reducers apply changes only to the appropriate site.

Reply:
The reducer for this is in #18148. Is that what you meant?